src/sudoku/singlarobin/react/component/Input/styles.module.css

```diff
@@ -0,0 +1,19 @@
+.wrapper {
+    background-color: transparent;
+    font-size: 1rem;
+    color: #000;
+    text-align: center;
+    outline: none;
+    border: 1px solid #000;
+    padding: 0.5rem;
+    margin: 0rem;
```

Comment:
is this necessary?

Reply:
padding and border is required, others are removed